src/TimeZones.jl

```diff
@@ -76,6 +77,22 @@ function TimeZone(str::AbstractString)
     end
 end
 
+
+"""
+    @tz_str str -> TimeZone
+
+Constructs the `TimeZone` of `str` with `TimeZone` function, which see.
```

Comment:
This was done on purpose
* https://en.wiktionary.org/wiki/q.v.
* https://www.urbandictionary.com/tags.php?tag=which%20see

I saw used often elsewhere, but if you think it's difficult to understand, please suggest me something more common (I'm not a native English speaker).